sockeye/arguments.py

```diff
@@ -430,11 +430,21 @@ def add_training_args(params):
                               help='Minimum number of epochs (passes through the training data) '
                                    'before fitting is stopped. Default: %(default)s.')
 
-    train_params.add_argument('--dropout',
+    train_params.add_argument('--embed-dropout',
+                              type=multiple_values(2, data_type=float),
+                              default=(.0, .0),
+                              help='Dropout probability for source & target embeddings. Use <val>:<val> to specify '
+                                   'separate values. Default: %(default)s.')
+    train_params.add_argument('--rnn-dropout',
+                              type=multiple_values(2, data_type=float),
+                              default=(.0, .0),
+                              help='RNN variational dropout probability for encoder & decoder RNNs.'
+                                   'Use <val>:<val> to specify separate values. Default: %(default)s.')
+
+    train_params.add_argument('--rnn-decoder-hidden-dropout',
                               type=float,
-                              default=0.,
-                              help='Dropout probability for source embedding and source and target RNNs. '
-                                   'Default: %(default)s.')
+                              default=.0,
+                              help='Dropout probability for hidden state in RNN decoder. Default: %(default)s.')
```

Comment:
'hidden state' + the "--rnn" prefix make it sound like it's related to the decoder RNN, how could we make it clear that this is for the hidden state that combines context with the upper hidden state?
For the description one could write "Dropout probability for hidden state that combines the context with the RNN hidden state in the decoder".